types/aws-lambda/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "@definitelytyped/dtslint/dt.json",
     "rules": {
-        "npm-naming": false
+        "npm-naming": false,
+        "void-return": false
```

Comment:
[DefinitelyTyped suggests](https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson) not to add any special blanket rules for packages. I agree that the rule shouldn't be triggering for our usecase as the suggested `undefined` replacement does not work when I tried locally.

With that in mind I think disabling it just as a one-off is probably the more sanctioned approach:
```ts
// tslint:disable-next-line:void-return
export type SQSHandler = Handler<SQSEvent, SQSBatchResponse | void>;
```

Reply:
Yeah, sounds good to me.